cmd/maya-apiserver/app/server/backup_endpoint_v1alpha1.go

```diff
@@ -390,7 +396,8 @@ func findLastBackupStat(clientset versioned.Interface, bkp *v1alpha1.CStorBackup
 		return v1alpha1.BKPCStorStatusFailed
 	}
 
-	// let's check if snapname matches with current snapshot name
+	// lastbkp stores the last(PrevSnapName) and 2nd last(SnapName) completed snapshot
```

Comment:
Can the variable be more clear like: 
- PrevSnapName
- PreviousToPrevSnapName 

Or

- LastSnapName
- SecondLastSnapName

Reply:
As of now Spec for CStorBackups and CStorCompletedBackups are same. So We can't rename this. We can create duplicate spec for that and rename those variables, keeping json tag same to avoid any upgrade.

Reply:
We can target this change in next cycle.